main.go

```diff
@@ -395,6 +405,66 @@ func NewExporter(server string, timeout time.Duration, logger log.Logger) *Expor
 			[]string{"slab"},
 			nil,
 		),
+		itemsNumberHot: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystemSlab, "items_number_hot"),
```

Comment:
We like to name things based on the "unit". The unit here seems to be "items". Also, "number" is a little redundant.

I would suggest this naming pattern:

```suggestion
			prometheus.BuildFQName(namespace, subsystemSlab, "hot_items"),
```

Reply:
Makes sense, I have followed the name of metrics reported by memcached in https://github.com/memcached/memcached/blob/master/doc/protocol.txt, your suggestion is definitely clearer.

Reply:
Yea, this is one of the older exporters, existing before we had the modern naming conventions. It looks like we need to do some cleanup of the code here.

Let's do the minimum work in this PR, and I can see about following up with some additional cleanup work.